bin/alluxio-start.sh

```diff
@@ -45,6 +45,7 @@ MOPT (Mount Option) is one of:
 -f  format Journal, UnderFS Data and Workers Folder on master
 -N  do not try to kill prior running masters and/or workers in all or local
 -w  wait for processes to end before returning
+-m  launch monitor process to ensure masters, workers and proxies come up.
```

Comment:
"... to ensure the target processes come up"

Reply:
Done